### PR TITLE
Create Sidekiq Worker for Emailing Tax Warnings

### PR DESCRIPTION
**Pull request by thczzz**

### CODE REVIEW

**Review by Snick555 — APPROVED** [review]

**Comment by thczzz** [comment]
Todo: Make it work in production

File: app/jobs/tax_mailer_job.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class TaxMailerJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    due_taxes = Tax.due_taxes
```
Comment (Snick555) [inline app/jobs/tax_mailer_job.rb:5]: why do you need it to set into variable?

File: app/models/tax.rb [hunk 2]
```diff
@@ -2,4 +2,9 @@ class Tax < ApplicationRecord
   belongs_to :car
   monetize :amount_cents, numericality: { in: 0.01..4294967295 }
   validates :description, length: {maximum: 350}
+
+  # Class method
+  def self.due_taxes
+    where("date + 365 = ?", 14.days.from_now.to_date)
```
Comment (Snick555) [inline app/models/tax.rb:8]: this logic is a little strange :) 
I would prefer to do it like 
```
DAYS_AGO_DUE_TAXES = 351
scope :due_taxes, -> { where(date: DAYS_AGO_DUE_TAXES.days.ago.to_date) }

**Comment by thczzz** [comment]
Todo: Deny access to /sidekiq for non-admin users in production.

**Review by apoltieva — APPROVED** [review]

**Pull request merged**